source/release_notes/6.0.2.rst

```diff
@@ -0,0 +1,52 @@
+.. _PSMDB-6.0.2-1:
+
+================================================================================
+*Percona Server for MongoDB* 6.0.2-1
+================================================================================
+
+:Date: September , 2022
+:Installation: `Installing Percona Server for MongoDB <https://www.percona.com/doc/percona-server-for-mongodb/6.0/install/index.html>`_
+
+
+We are pleased to announce the availability of Percona Server for MongoDB (PSMDB) 6.0.2-1 – the new major version of the source available, drop-in replacement of MongoDB 6.0 Community edition. It is available for `download from Percona website <https://www.percona.com/downloads/percona-server-mongodb-6.0/#>`_ and for `installation from Percona Software Repositories <https://www.percona.com/doc/percona-server-for-mongodb/6.0/install/index.html>`_.
+
+PSMDB 6.0.2-1 fully supports MongoDB 6.0 protocols and drivers and does not require any code modifications.
+
+PSMDB 6.0.2-1 includes all the features of MongoDB 6.0 Community Edition. These include enhancements to the existing features, plus the bunch of new ones. 
+
+Among new features are the following:
```

Comment:
I do not see anything about queryable encryption :( and the basic mode (explicit) is AFAIK included in CE

Reply:
It's been decided not to include it in the RN